app/assets/stylesheets/shared/header.scss

```diff
@@ -1,18 +1,17 @@
 .header {
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.